### PR TITLE
Update task list in README

### PR DESCRIPTION
**Pull request by coopy**
Based on `builder help builder-react-component`; cleaned up [ROOT] references.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -78,125 +78,140 @@ settings](config/babel/.babelrc).
 
 ```
 $ builder help builder-react-component
-[builder:help]
 
 Usage:
 
-  builder [action] [task]
+  builder builder-react-component [flags] [task]
```
Comment (coopy) [inline README.md]: Should this be `builder [action] [flags] [task]`?

Comment (ryan-roemer) [inline README.md]: @coopy -- Yep. Upstream PR for this at https://github.com/FormidableLabs/builder/pull/52

File: README.md [hunk 2]
```diff
@@ -78,125 +78,140 @@ settings](config/babel/.babelrc).
...
+    npm run builder:lint-server
+
+  builder:lint-server
+    eslint --color -c config/eslint/.eslintrc-server config/karma config/webpack
+
+  builder:prepublish
+    node util/dev.js
+
   npm:postinstall
-    [builder-react-component] node -e "require('fs').stat('lib', function(e,s){process.exit(e || !s.isDirectory() ? 1 : 0)})" || builder run build-lib
```
Comment (ryan-roemer) [inline README.md:94]: Hmmm... These shouldn't need to be manually edited.

Not a blocking comment for now, but food for thought for https://github.com/FormidableLabs/builder/issues/12

Comment (coopy) [inline README.md:94]: I was going off of the `npm run` example that lists all tasks. Should the archetype name be included in brackets or is this format OK (listing just the task contents)?

Comment (ryan-roemer) [inline README.md:94]: I think keep the archetype name for now until we decide something more canonical?

Comment (curiouslychase) [inline README.md:94]: At the point that we start allowing "multiple" archetypes (if that happens), having the archetype in `[ ]` is a really great debugging tool.

Comment (coopy) [inline README.md:94]: When issuing `builder help builder-react-component` in the `builder-react-component` project, the task list format has `[ROOT]` prefixes, not `[builder-react-component]`.

Comment (ryan-roemer) [inline README.md:94]: @coopy -- `builder` is not currently meant to run straight up in an Archetype repo. See: https://github.com/FormidableLabs/builder-react-component/blob/master/CONTRIBUTING.md#documentation

This is part of what we'll tackle "for realz" in https://github.com/FormidableLabs/builder/issues/12

**Comment by coopy** [comment]
@ryan-roemer ready for re-review.

**Comment by ryan-roemer** [comment]
@coopy -- This is superseded by https://github.com/FormidableLabs/builder-react-component/pull/21 (apologies for the runaround...)

**Comment by coopy** [comment]
No worries!

**Pull request closed without merging**